deepmd/pt/loss/property.py

```
@@ -21,9 +24,13 @@ class PropertyLoss(TaskLoss):
     def __init__(
         self,
         task_dim,
+        var_name: str,
         loss_func: str = "smooth_mae",
         metric: list = ["mae"],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)